### PR TITLE
Remove row_based parameter in bulk_insert tests

### LINKED ISSUE #20170: [Enhancement]: bulk_insert tests shall be updated for row_based parameter not required any more
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What would you like to be added?

row_based is not required in bulk_insert()
stop numpy supported 

### Why is this needed?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by yanliang567**
Resolve: #20170 
Signed-off-by: yanliang567 <yanliang.qiao@zilliz.com>

### CODE REVIEW

File: tests/python_client/bulk_insert/test_bulk_insert.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def teardown_method(self, method):
 class TestBulkInsert(TestcaseBaseBulkInsert):
 
     @pytest.mark.tags(CaseLabel.L3)
-    @pytest.mark.parametrize("is_row_based", [True, False])
+    @pytest.mark.parametrize("is_row_based", [True])
```
Comment (binbinlv) [inline tests/python_client/bulk_insert/test_bulk_insert.py:76]: why not remove this parameter directly?

Comment (yanliang567) [reply tests/python_client/bulk_insert/test_bulk_insert.py:76]: it used for generating data files, and i assume that we would support column_based in the coming future

Comment (binbinlv) [reply tests/python_client/bulk_insert/test_bulk_insert.py:76]: OK

**Pull request merged**